Fixed SQL Injection

1ue999 opened the pull request:
this should work (but please test EVERYTHING and fix it if possible)

1ue999 committed:
Fixed SQL Injection

BnDLett commented:
Code looks good. I'll merge it into a new branch and I'll work on the code from there.

BnDLett commented:
IF I can figure out how to pull that off in GitHub

BnDLett committed:
Fixed version number.

BnDLett committed:
Aded features and bug fixes

Added:
- The remaining time for a ban now appears in the ban message.
- `getBanEndTime` now has documentation.
- You can now supply a username and password for the database.

Fixed:
- A leaving player is now removed from the player identification list.
- Cleaned up and fixed `getBanStartTime` documentation.
- `removeBan` had `'%s'` in the statement instead of `?`
- Inconsistent styling among the

BnDLett reviewed (approved):
All changes are good. Can be merged to master branch.
